app/src/main/kotlin/org/fossify/phone/fragments/FavoritesFragment.kt

```diff
@@ -183,9 +183,19 @@ class FavoritesFragment(context: Context, attributeSet: AttributeSet) : MyViewPa
             val orderIds = items.map { it.contactId }
             val orderGsonString = Gson().toJson(orderIds)
             config.favoritesContactsOrder = orderGsonString
+            assignCustomOrderToAllContact(orderIds)
```

Comment:
```suggestion
            allContacts = ArrayList(items)
```

This is all you need.